CHANGES.rst

```diff
@@ -33,6 +33,7 @@ New Features
 - ``astropy.nddata``
 
 - ``astropy.stats``
```

Comment:
I did a test using pep8. Worked fine, but travis-ci is still complaining about the scipy import.